src/m365/spo/commands/tenant/tenant-recyclebinitem-remove.spec.ts

```diff
@@ -39,7 +33,6 @@ describe(commands.TENANT_RECYCLEBINITEM_REMOVE, () => {
   beforeEach(() => {
     const futureDate = new Date();
     futureDate.setSeconds(futureDate.getSeconds() + 1800);
```

Comment:
If you've removed the stubs, can these unused variable not be removed either?

Reply:
It makes the tests fail with 'invalid request'
![image](https://github.com/pnp/cli-microsoft365/assets/35696168/0e316487-6ddb-46ca-a5da-b3547ab51169)

Reply:
yeah, mixed it up here. This comment was for the review comment above this one 😄